<% if obj.errors.any? %>
    <div id="error_explanation">
      <h3><%= obj.errors.count %> 箇所のエラーにより、正しく処理が行えませんでした:</h3>

      <ul>
      <% obj.errors.full_messages.each do |message| %>
        <li><%= message %></li>
      <% end %>
      </ul>
    </div>
<% end %>